index.html

```diff
@@ -45,7 +45,7 @@ <h2 class="subtitle bold"><q>Hello! Sharing world</q></h2>
                 <p class="description">
                     A joyful celebration in the value of openess and sharing,
                     creating a positive change with people from over 10 countries
-                    is taking place in July, in Benin.
+                    is taking place in July, in Cotonou, Benin.
                 </p>
                 <div class="hero-event">
```

Comment:
### I write to you the review for this part at these lines here as I can not see all the code of the page 😊

- [ ] While you make a great effort😊 I see that you forgot to use the article tags for this section, so it will be better if you use them as this from the requirements, so for this section💯😃, It contains a bottom section with a column that includes a section header, a divider, and six individual articles, each with an image, title, subtitle, and description. Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description..🚀👌


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your code of design  | The design | 
| --- | --- |
|![image](https://github.com/lily4178993/leaders-week/assets/83360636/f13135d3-db8f-451e-b879-1099e859c6e5) |     ![image](https://github.com/lily4178993/leaders-week/assets/83360636/62520848-2163-449c-ba04-b9cb2d016a9a) |